src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -208,6 +208,17 @@ public function prepend(ContainerBuilder $container)
                 ]
             );
         }
+
+        if ($container->hasExtension('framework')) {
+            $container->prependExtensionConfig(
+                'framework',
+                [
+                    'cache' => [
+                        'directory' => '%sulu.common_cache_dir%',
```

Comment:
Is it possible that this is not the only place where we have to fix this? It is still creating some cache entries in the `admin` and `website` cache folders for me. Actually it seems to me like only the collaborations created by Sulu are stored in the `common` folder this way.

Reply:
FYI: I tested by just running `bin/console server:run` in sulu/sulu.

Reply:
Currently it seems only the `cache.app` which use the `cache.adapter.filesystem` this directory is used and set correctly here:

https://github.com/symfony/symfony/blob/bfe6b6fb41b677a35e7fbc89fcc7d9382be0c41b/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L1806-L1807

The cache directory for the `system` cache is not replaced as you see and is hardcoded in:

https://github.com/symfony/symfony/blob/bfe6b6fb41b677a35e7fbc89fcc7d9382be0c41b/src/Symfony/Bundle/FrameworkBundle/Resources/config/cache.xml#L49

Not sure if this is a bug on symfony side.

Reply:
I created a PR on symfony side lets see what the will do https://github.com/symfony/symfony/pull/36378. But it has no affect on us as we don't use the `cache.system`.